google.json

```diff
@@ -780,7 +964,7 @@
     {
         "id": "com.google.android.contacts",
         "label": "Contacts",
-        "description": "Contacts by Google LLC.\nOccasionally runs in the background.",
+        "description": "Contacts by Google LLC.\nOccasionally runs in the background.\nIf you looking for alternative, you can try: Simple Contacts Pro, OpenContacts.",
```

Comment:
We don't suggest stuffs in the description. There's a dedicated section for it (which is already filled).